internal/dto/env.go

```diff
@@ -2,23 +2,21 @@ package dto
 
 import (
 	"fmt"
-	"go.arcalot.io/log"
 	"os"
+
+	"go.arcalot.io/log"
 )
 
-func LookupEnvVar(key string, logger log.Logger) Verbose {
+// LookupEnvVar returns an error if an environment variable is not set or empty.
+// The error includes the registry url to destinguish which registry encountered the error.
```

Comment:
This comment still only describes the validation. Not that it actually returns the value from the env variable.